optimum/exporters/tasks.py

```diff
@@ -931,7 +937,14 @@ class TasksManager:
             onnx="YolosOnnxConfig",
         ),
     }
-    _UNSUPPORTED_CLI_MODEL_TYPE = {"unet", "vae-encoder", "vae-decoder", "clip-text-model", "trocr"}
+    _UNSUPPORTED_CLI_MODEL_TYPE = {
```

Comment:
Could you add an explicit check in `main_export` with meaningful error?

Reply:
done in [`6c122`](https://github.com/huggingface/optimum/pull/1168/commits/6c12200d33e7c9451fc1afa59f6251aeed6c3205)